init.sh

```diff
@@ -2,138 +2,92 @@
 
 ok="Ok, going to the next question."
 
-cat << "EOF"
+read -r -d '' dna_ascii << "EOF"
```

Comment:
Saving the DNA logo thingie into a variable (dna_ascii) to minimise duplication

Reply:
Getting rid of brackets and commas/collons is why I prefer shell.
But, if this adds compatibility with other shells than BASH, ok.